src/main/java/pacApp/pacSoapConnector/context/TestService2.java

```diff
@@ -0,0 +1,15 @@
+package pacApp.pacSoapConnector.context;
+
+import org.springframework.stereotype.Service;
+
+import pacApp.pacSoapConnector.SoapConvertCurrencyConnector;
+
+@Service
+public class TestService2 implements ISoapTestService{
+
+	@Override
+	public String runTest() {
+		new SoapConvertCurrencyConnector().convertCurrency();		
+		return null;
```

Comment:
return null? I guess this test is no finished?

Reply:
Is a test class. The class is now deleted.